<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grafo_window</class>
 <widget class="QDialog" name="grafo_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_insert_vertice">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Insertar Vertice</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_vertice">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_vertice1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_vertice2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_distancia">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertice 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertice 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Distancia</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_insert_arista">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Insertar Arista</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_picture">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>771</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_test">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>490</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_dijsktra">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Dijsktra</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_origen_dj">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Origen</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_destino_dj">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Destino</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_BFS">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>BFS</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_picture2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>460</y>
     <width>631</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>480</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BFS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>550</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DFS</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_picture3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>530</y>
     <width>631</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_DFS">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>DFS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
